### ISSUE TITLE
[請求] 新增（符合 GH 的）許可證文件

### ISSUE THREAD

**Issue by MarcoRavich** [as2648as/AIO_Optimizer#1]
**[Google-CHI]**
您好，首先感謝您所做的出色工作！

因為我已經在 [Opendows 專案](https://forart. it/opendows)，您能為它新增一個符合 GH 的許可證檔案嗎？

如您所知，明確的授權條款對於讓任何人更好/更快地了解如何在其他開放專案中重複使用/改編/修改來源（而不僅僅是）非常重要，反之亦然。

儘管這聽起來像是一個小問題，但許可證文件的遺漏顯然也會導致相關徽章的生成不一致：

![](https://badgen.net/github/license/as2648as/AIO_Optimizer)
（徽章產生器 URL：https://badgen.net/github/license/as2648as/AIO_Optimizer)

您可以透過 GH 的許可證精靈工具輕鬆設定標準化許可證。

最後但並非最不重要的一點是，我邀請您加入 [CS 討論](https://github.com/mewostick/Creosynth/discussions) 分享有關 Windows 調整的想法。

希望有幫助/啟發！

**[ENG]**
Hi there, 1st of all thanks for this awesome work !

Since I've 'doxed' it in the [TWEAKERS page](https://github.com/MarcoRavich/Opendows/blob/main/Tweakers.md#-) of the [Opendows project](https://forart.it/opendows), can you please add a GH-compliant license file for it ?

As you know, expliciting licensing terms is extremely important to let anyone better/faster understand how to reuse/adapt/modify sources (and not only) in other open projects and vice-versa.

Although it may sounds like a minor aspect, license file omission obviously causes an inconsistent generation of the relative badge too:

![](https://badgen.net/github/license/as2648as/AIO_Optimizer)
(badge-generator URL: https://badgen.net/github/license/as2648as/AIO_Optimizer)

You can easily set a standardized one through the GH's license wizard tool.

Last but not least, I invite you to join [CS Discussions](https://github.com/mewostick/Creosynth/discussions) to share ideas about Windows tweakings.

Hope that helps/inspires !

**Comment by as2648as** [comment]
[Copilot-Translate]

Hi @MarcoRavich, Thank you very much for your attention and notice. 

Since this project is for my personal use, the text is entirely in Chinese. Additionally, I only write or paste content that I understand. 

I have limited familiarity with open-source licenses, but I chose the MIT license because it is one of the most permissive licenses available.

I won't be updating it for a long time because this project is typically only used and adjusted when I reinstall my system. However, this doesn't mean I won't use it. If there are any new features, I will still update it. 

Finally, I appreciate the opportunity and will try to participate in the discussion when I have some free time.  :)
